src/main/java/com/ericsson/ei/frontend/model/BackEndInformation.java

```diff
@@ -41,6 +42,7 @@
 
     @Value("${ei.useSecureHttpBackend:#{false}}")
     @JsonProperty("https")
+    @SerializedName("https")
```

Comment:
good, but why only for one field?

Reply:
Why do we need this?

Reply:
@JsonProperty("https") - this for jackson
@SerializedName("https") - for gson

because only in this case field name and serialized name different